game/scripts/npc/abilities/zuus_cloud.txt

```diff
@@ -8,6 +8,9 @@
     "AbilityBehavior"                                     "DOTA_ABILITY_BEHAVIOR_POINT | DOTA_ABILITY_BEHAVIOR_AOE | DOTA_ABILITY_BEHAVIOR_HIDDEN | DOTA_ABILITY_BEHAVIOR_SHOW_IN_GUIDES"
     "AbilityUnitDamageType"                               "DAMAGE_TYPE_MAGICAL"
     "SpellImmunityType"                                   "SPELL_IMMUNITY_ENEMIES_NO"
+    "BaseClass"                                           "ability_lua"
+    "ScriptFile"                                          "abilities/oaa_zuus_cloud.lua"
```

Comment:
Create a completely new ability called `zuus_cloud_oaa` and change Zeus' ability KV. Overwriting default abilities is bad practice as it can sometimes still have hardcoded behaviour. Change the tooltip to have a note about not being affected by CDR while you're at it, since you'll need to also copy the tooltip.

The naming convention for the Lua file is usually to not have the hero name, but maybe this should be an exception since "oaa_cloud" is not really a great name.

Reply:
this is already an overrid of a dota abiltiy, but I got your point.

Reply:
I tried to create a new zuus_cloud_oaa but it is not showing when scepter I have scepter.
maybe you could try something but it did not worked for me.
probably some valve hardcoded bullshit